stylesheets/modules/_footer.scss

```diff
@@ -48,32 +40,17 @@
   margin-top: 60px;
   padding-top: 45px;
   border-top: 1px solid rgba($white, .1);
-  @media (min-width: 1040px) {
-    margin: {
-      right: auto;
-      left: auto;
-    }
-    max-width: 940px;
-  }
 }
 
 .footer__sponsors {
-  margin: {
-    right: auto;
-    left: auto;
-  }
-  @media (max-width: 579px) {
-    width: 78%;
-  }
-  @media (min-width: 580px) and (max-width: 1039px) {
-    width: 90%;
-  }
+	margin: 0 2%;
```

Comment:
It looks like the tabbing is odd in some areas as the default is 2 spaces but most of changes have 4 spaces as a tab (eg. Line 53, Line 173-196).

Reply:
thank you, I could fix it